slycot/math.py

```diff
@@ -18,11 +18,113 @@
 #       MA 02110-1301, USA.
 
 from . import _wrapper
-from .exceptions import raise_if_slycot_error
+from .exceptions import SlycotParameterError, raise_if_slycot_error
 
 import numpy as np
 
 
+def mb02ed(typet: str,T: np.ndarray, B: np.ndarray, n: int, k: int,  nrhs: int):
+    """ X, T = mb02ed(typet, T, B, n, k, nrhs)
+
+    Solve a system of linear equations T*X = B or X*T = B with a positive
+    definite block Toeplitz matrix T.
+
+    Parameters
+    ----------
+    typet: str
+        Specifies the type of T:
+            - 'R': T contains the first block row of an s.p.d. block Toeplitz matrix,
+                and the system X*T = B is solved.
+            - 'C': T contains the first block column of an s.p.d. block Toeplitz matrix,
+                and the system T*X = B is solved.
+        Note: the notation x / y means that x corresponds to
+                typet = 'R' and y corresponds to typet = 'C'.
+    T : ndarray
+            The leading k-by-n*k or n*k-by-k part of this array must contain the first
+            block row/column of an s.p.d. block Toeplitz matrix.
+    B : ndarray
+            The leading nrhs-by-n*k or n*k-by-nrhs part of this array must contain the
+            right-hand side matrix B.
+    n : int
+            The number of blocks in T. n >= 0.
+    k : int
+       The number of rows/columns in T, equal to the blocksize. k >= 0.
+    nrhs : int
+            The number of right-hand sides. nrhs >= 0.
+
+    Returns
+    -------
+    X : ndarray
+        Leading nrhs-by-n*k / n*k-by-nrhs part of
+        this array contains the solution matrix X.
+    T: ndarray
+        If no error is thrown  and  nrhs > 0,  then the leading
+        k-by-n*k / n*k-by-k part of this array contains the last
+        row / column of the Cholesky factor of inv(T).
+
+    Warns
```

Comment:
Change to `Raises` so that the docstring parser will generate the correct exception.

https://github.com/python-control/Slycot/blob/60a5c71190d790b18d62c08b8685e289d75ddf94/slycot/exceptions.py#L134